tests/test_data_handler.py

```diff
@@ -3,6 +3,7 @@
 """
 
 import os
+os.environ["ZFIT_DISABLE_TF_WARNINGS"] = "1" # pylint: disable=wrong-import-position
```

Comment:
maybe we could add it in the `DataHandler` and in the `F2MassFitter`, what do you think?